common/persistence/sql/sqlplugin/mysql/session/session.go

```diff
@@ -60,6 +60,7 @@ const (
 
 var (
 	errMySQL8VisInterpolateParamsNotSupported = errors.New("interpolateParams is not supported for mysql8 visibility stores")
+	errUnspecifiedMySQLVersion                = errors.New("bug: mysql version not specified")
```

Comment:
Is this unused?

Reply:
Yeah, good catch. This is left-over from when I manually checked MySQL versions as parameter implementation works fine in MySQL 5.7 visibility stores